src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough8.java

```diff
@@ -14,15 +14,15 @@ void methodLastLine(int i) {
             switch (i) {
                 case 0:
                     i++;
-                    /* block */ // comment
-                    // fall thru
+                 // block
+                 // fall thru
```

Comment:
Please do not change existing test input code.
Change violation comments only.
If not possible, please share problem

Reply:
@romani Done

Reply:
Please restore indentation, revert all charges in the this lines